perun-openapi/openapi.yml

```diff
@@ -194,6 +201,8 @@ components:
         sortColumn: { $ref: '#/components/schemas/MembersOrderColumn' }
         searchString : { type: string }
         statuses: { type: array, items: { $ref: '#/components/schemas/VoMemberStatuses' } }
+        group: { $ref: '#/components/schemas/Group' }
+        groupStatuses: { type: array, items: { $ref: '#/components/schemas/MemberGroupStatus' } }
```

Comment:
Also, there is a file `perun-utils/rpc-methods-javadoc-generator/parseRpcMethods.pl` which contains example parameters for our API methods. There is a `$objectExamples{"MembersPageQuery"}`. Please, update this example as well.

Reply:
Updated.